modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -153,6 +153,16 @@ These adjustments are only applied to cleartext, edge-terminated, and re-encrypt
 
 For request headers, these adjustments are applied only for routes that have the `haproxy.router.openshift.io/h1-adjust-case=true` annotation. For response headers, these adjustments are applied to all HTTP responses. If this field is empty, no request headers are adjusted.
 
+|`tuningOptions`
+|`tuningOptions` specifies options for tuning the performance of Ingress Controller pods.
+
+* `headerBufferBytes` specifies how much memory is reserved in bytes for Ingress Controller connection sessions. This value must be at least `16384` if HTTP/2 is enabled for the Ingress Controller. If this field is empty, the Ingress Controller uses a default value of `32768` bytes. Setting this field is optional and not recommended because `headerBufferBytes` values that are too small can break the Ingress Controller, and `headerBufferBytes` values that are too large could cause the Ingress Controller to use significantly more memory than necessary.
```

Comment:
nitpicky ideas, feel free to disregard if I miss the mark.

* maybe "...how much memory, in bytes, to reserve..."
* Q: is the memory reserved for each connection (my guess) or all connections?
* Q: any way to be more precise than "break"?  Are new connections refused?
* Way up the HTML page, I spotted "If not set, the default value is 2." for replicas, so maybe for consistency, "If this field is not set, the Ingress..."
* The "not recommended" catches my eye.  Would it be OK to replace with "Use care when you set this field because..."? Or, "Be aware that specifying a value that is too small..."

I think these comments apply to the foll two bullets as well.